src/schema/city/index.ts

```diff
@@ -201,7 +204,7 @@ const nearestCity = (latLng: LatLng) => {
 }
 
 const citiesOrderedByDistance = (latLng: LatLng): Point[] => {
-  let cities: Point[] = Object.values(cityData)
+  let cities: Point[] = Object.values(cityDataSortedByDisplayPreference)
```

Comment:
Are we deliberately calling  `Object.values` on an array here? Looked odd to me, but the REPL says that actually works 🤷‍♂️

Reply:
Hah, wowzers! So we learn.

I guess TS realised that it was fine and that’s why I missed it too. Even though it works, I’ll send a lil cleanup PR in a bit.